src/Entities/Address.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TrueLayer\Entities;
+
+use TrueLayer\Interfaces\AddressInterface;
+
+class Address extends Entity implements AddressInterface
+{
+    /**
+     * @var string
+     */
+    protected string $addressLine1;
+
+    /**
+     * @var string
+     */
+    protected string $addressLine2;
+
+    /**
+     * @var string
+     */
+    protected string $city;
+
+    /**
+     * @var string
+     */
+    protected string $state;
+
+    /**
+     * @var string
+     */
+    protected string $zip;
+
+    /**
+     * @var string
+     */
+    protected string $countryCode;
+
+    /**
+     * @var string[]
+     */
+    protected array $arrayFields = [
+        'address_line1',
+        'address_line2',
+        'city',
+        'state',
+        'zip',
+        'country_code',
+    ];
+
+    /**
+     * @var string[]
+     */
+    protected array $rules = [
+        'address_line1' => 'string|required',
+        'address_line2' => 'string|nullable',
+        'city' => 'string|required',
+        'state' => 'string|required',
+        'zip' => 'string|required',
+        'country_code' => 'string|required',
+    ];
+
+    /**
+     * @return string
+     */
+    public function getAddressLine1(): string
+    {
+        return $this->addressLine1;
```

Comment:
Could we fallback to `?? null;` for all fields? We don't know if the api will always return a value for them.